### PR TITLE
Add `New Featured Artist: cast heal` news post

### PR DESCRIPTION
**Pull request by pishifat**
!!!

### CODE REVIEW

File: news/2024/2024-09-07-new-featured-artist-cast-heal.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+date: 2024-09-07 20:00:00 +0000
+---
+
+Is **cast heal** our newest Featured Artist? We'll let you figure it out.
+
+![](https://assets.ppy.sh/artists/436/header.jpg)
+
+When we asked **cast heal** for some of their achievements and milestones for this article, they responded with "filthy runoff", so that should give you a solid idea of what you're getting into.
+
+**x** melancholic drum and bass tracks are officially part of [**cast heal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/436), all of which are pre-timed so your impulsive mapping decisions won't be blocked by anything.
```
Comment (Walavouchey) [inline news/2024/2024-09-07-new-featured-artist-cast-heal.md:13]:
```suggestion
**10** melancholic drum and bass tracks are officially part of [**cast heal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/436), all of which are pre-timed so your impulsive mapping decisions won't be blocked by anything.
```

File: news/2024/2024-09-07-new-featured-artist-cast-heal.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Experience [the map from the video above](https://osu.ppy.sh/beatmapsets/2217395) hosted by [Okoratu](https://osu.ppy.sh/users/1623405)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/436/run%20to%20her/cast%20heal%20-%20run%20to%20her.mp3">
+</audio>
+
+### cast heal - lasting entropy
+
+Or try [the first ranked **cast heal** map](https://osu.ppy.sh/beatmapsets/2134803) also hosted by [Okoratu](https://osu.ppy.sh/users/1623405)!
```
Comment (Walavouchey) [inline news/2024/2024-09-07-new-featured-artist-cast-heal.md:33]: we usually capitalise ranked the category

```suggestion
Or try [the first Ranked **cast heal** map](https://osu.ppy.sh/beatmapsets/2134803) also hosted by [Okoratu](https://osu.ppy.sh/users/1623405)!
```

File: news/2024/2024-09-07-new-featured-artist-cast-heal.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+If neither of those work for you, try [this one](https://osu.ppy.sh/beatmapsets/2183856) hosted by [you-know-who](https://osu.ppy.sh/users/1623405).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/436/run%20to%20her/cast%20heal%20-%20run%20to%20her.mp3">
+</audio>
+
+---
+
+Whatever's on your mind is nowhere near as important as visiting [**cast heal**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/436) and starting on your next map. Your professors are reading this announcement too, so they'll understand why your assignments are late.
```
Comment (Walavouchey) [inline news/2024/2024-09-07-new-featured-artist-cast-heal.md:49]: good advice 👍

File: news/2024/2024-09-07-new-featured-artist-cast-heal.md (hunk 1 above)
Comment (pishifat) [reply news/2024/2024-09-07-new-featured-artist-cast-heal.md:13]: one track was added at the last minute oops

File: news/2024/2024-09-07-new-featured-artist-cast-heal.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/436/lasting%20entropy/cast%20heal%20-%20lasting%20entropy.mp3">
+</audio>
+
+### cast heal - fragments
+
+If neither of those work for you, try [this one](https://osu.ppy.sh/beatmapsets/2183856) hosted by [you-know-who](https://osu.ppy.sh/users/1623405).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/436/run%20to%20her/cast%20heal%20-%20run%20to%20her.mp3">
```
Comment (Walavouchey) [inline news/2024/2024-09-07-new-featured-artist-cast-heal.md:44]: wrong link

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**